packages/host/app/styles/app.css

```diff
@@ -513,11 +548,26 @@ nav > .directory-level {
   pointer-events: auto;
 }
 
+.operator-mode-card-stack__card--cards-grid {
```

Comment:
`--` notation is used for describing a state or a version of the element. `cards-grid` is a block so this brings some confusion

Reply:
In this case, cards-grid is a state because I want to differentiate the styles of the cards grid and other cards in operator mode.